src/ontology/mondo-ingest.Makefile

```diff
@@ -373,6 +373,7 @@ USE_MONDO_RELEASE=false
 
 # Builds tmp/mondo/ and rebuilds mondo.owl and mondo.sssom.tsv, and stores hash of latest commit of mondo repo main branch in tmp/mondo_repo_built
 tmp/mondo_repo_built:
+	git config --global --add safe.directory /work/src/ontology/tmp/mondo
```

Comment:
## About the bug & the fix
@twhetzel @matentzn This is a weird bug. I don't know if this is only causing a problem for the synonym sync. It's the only time the bug has appeared. But at the same time, I don't know why it would only be related to the synonym sync.

We have a meeting coming up shortly; I think it will be easier to discuss there.